packages/appstore/src/components/appstore-banner/svg-migration-banner.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Button, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import AppstoreBannerImage from 'Assets/svgs/banner';
+
+const SVGMigrationBanner = observer(() => {
+    const {
+        ui: { is_mobile },
+    } = useStore();
+    //TODO: Will add condition check for svg deriv mt5 and svg financial mt5 when API is ready
+    const is_deriv_mt5 = true;
+    const is_financial_mt5 = true;
+    const is_deriv_financial_mt5 = is_deriv_mt5 && is_financial_mt5;
+    const image = is_mobile ? 'svg_migrate_mobile' : 'svg_migrate_desktop';
+    const size: string = is_mobile ? 'xs' : 'm';
+
+    const getAccountTitle = () => {
+        if (is_deriv_mt5) return 'MT5 Derived SVG';
+
+        if (is_financial_mt5) return 'MT5 Financial SVG';
```

Comment:
here, shall we simply write a return statement without check?
like, one of them is the `default` case?